### PR TITLE
Add tox-node module

### PR DESCRIPTION
**Pull request by suhr**

### CODE REVIEW

File: modules/services/tox-node.nix [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+      after = [ "network.target" ];
+      wantedBy = [ "multi-user.target" ];
+
+      serviceConfig = {
+        ExecStart = "${pkgs.tox-node}/bin/tox-node --config ${configFile}";
+        Restart = "always";
+      };
+    };
+  };
+}
```
Comment (kpp) [inline modules/services/tox-node.nix]: new line

File: modules/services/tox-node.nix [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  options.services.tox-node = {
+    enable = mkEnableOption "Tox Node service";
+
+    logType = mkOption {
+      type = types.string;
+      default = "Stderr";
+    };
+    keysFile = mkOption {
+      type = types.string;
+      default = "/var/lib/tox-node/keys";
```
Comment (kpp) [inline modules/services/tox-node.nix:98]: Do you have permissions to create this file? Did you create directory `/var/lib/tox-node`?

File: modules/services/tox-node.nix [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+    systemd.services.tox-node = {
+      description = "Tox Node";
+
+      after = [ "network.target" ];
+      wantedBy = [ "multi-user.target" ];
+
+      serviceConfig = {
+        ExecStart = "${pkgs.tox-node}/bin/tox-node --config ${configFile}";
+        User = "tox-node";
```
Comment (kpp) [inline modules/services/tox-node.nix:140]: What if there is no such user?

Comment (suhr) [reply modules/services/tox-node.nix:140]: It is created by `users.users."tox-node" = { ... }`

**Pull request merged**